src/interactivity/win32/Clipboard.cpp

```diff
@@ -515,7 +275,9 @@ void Clipboard::CopyTextToSystemClipboard(const TextBuffer::TextAndColor& rows,
 
     if (fAlsoCopyHtml)
     {
-        std::string HTMLToPlaceOnClip = GenHTML(rows);
+        const auto& fontData = ServiceLocator::LocateGlobals().getConsoleInformation().GetActiveOutputBuffer().GetCurrentFont();
+        int const iFontHeightPoints = fontData.GetUnscaledSize().Y * 72 / ServiceLocator::LocateGlobals().dpi;

```

Comment:
As far as the size is concerned:

When we're using the GDI renderer, the font size is in pixels, so this conversion is necessary. However, we can also use the DX renderer with conhost now, and it uses the font size as points. So it might not need this conversion. 

I don't really understand font points vs pixels though, @miniksa is the expert on that. I'd just make sure to test this out with both the GDI renderer and the DX renderer, and check that the generated HTML is a _reasonable_ size.

Reply:
The goal of this was to make the HTML look the same as what was being displayed on the primary renderer. To that end, if you need to get the point size of what's currently being displayed, then it should be a `Get` method on the renderer. 

This might mean you need to add `IRenderEngine::GetCurrentFontHeightInPoints()` and implement it appropriately to return the sizes. It's better than trying to reverse engineer it from the global font/dpi/unscaled sizes.

Reply:
Verified HTML text is copied at proper size.